index.d.ts

```diff
@@ -95,7 +95,6 @@ declare namespace suitest {
 		authContext: AuthContext;
 		appContext: Context;
 		pairedDeviceContext: Context;
-		testContext: Context;
```

Comment:
Removing `testContext` is a breaking change, we should increase suites-js-api version up to 2.0.0, or leave it as it is, but mark it as deprecated. @AntonSBilan @andriichumak

Reply:
we dont, but user may use it, because this is part of public api.

Reply:
I think we need just mark it as deprecated and then remove it in scope of refactoring.

Reply:
I cannot find it's mentioning in docs, probably it was public just for testing

Reply:
Anyway it will stop working after removing testName, so it will break the user logic if such exists

Reply:
This was never meant for users, I think this is a mistake that we display context variables in TS definitions at all.
We can leave it here for now, but please do mark both "testContext" and `startTest` with arguments as deprecated. We will remove it when 2.0 is released.

Reply:
But how should 'testContext' work, without testName there will be no data for it?

Reply:
Keep all the logic as it is in current version of the JS API. Update only two things:
1. Add deprecation annotations to the TS and allow for not passing test name if user do not want to.
2. Make sure test name and props are not sent to server, even if user defined them and testContext is created.